copier.yml

```
@@ -10,7 +10,7 @@ _envops:
   variable_start_string: "{{"
 
 # Other Copier configurations
-_min_copier_version: "4.1.0"
+_min_copier_version: "6.2.0"
```

Now that you're updating, do it more 😋 

```suggestion
_min_copier_version: "7.0.0"
```